optimum/amd/ryzenai/configuration.py

```diff
@@ -122,7 +601,7 @@ def __init__(
     ):
         super().__init__()
         self.opset = opset
-        self.quantization = self.dataclass_to_dict(quantization)
+        self.quantization = quantization.to_diff_dict() if quantization is not None else None
```

Comment:
Why only the diff? I think it is dangerous not to write the full config here, as defaults are not guaranteed to not change in the future.